pytest_reana/fixtures.py

```diff
@@ -472,21 +480,29 @@ def sample_yadage_workflow_in_db(app, default_user, session, yadage_workflow_wit
 
 
 @pytest.fixture()
-def sample_serial_workflow_in_db(app, default_user, session, serial_workflow):
+def sample_serial_workflow_in_db(
+    app, default_user, session, serial_workflow, sample_workflow_workspace
+):
     """Create a sample workflow in the database.
 
     Scope: function
 
     Adds a sample serial workflow in the DB.
     """
+    from reana_db.models import Workflow
```

Comment:
Moved REANA-DB imports form global scope to the fixtures to make REANA-Client pass on Python2.7 builds, [currently failing because of this](https://github.com/reanahub/reana-client/pull/429).

Reply:
I think you're missing `Base` import in `app` fixture.